src/raft.c

```diff
@@ -889,6 +946,8 @@ static int raftApplyLog(raft_server_t *raft, void *user_data, raft_entry_t *entr
         case RAFT_LOGTYPE_DELETE_UNLOCK_KEYS:
             unlockDeleteKeys(rr, entry);
             break;
+        case RAFT_LOGTYPE_END_SESSION:
+            handleEndClientSession(rr, entry);
```

Comment:
```suggestion
            handleEndClientSession(rr, entry);
            break;
```

Let's add this not to see a surprise later